### DISCUSSION TITLE
[Flutter user study opportunity] Multi-platform Development (10/21 - 11/5)

### DISCUSSION THREAD

**Discussion by jackkim9** [flutter/uxr#77] [General]
Have you published an app in multiple platforms (e.g., mobile+web, mobile+web+desktop, desktop+mobile)? Or are you actively developing an app targeting multiple platforms? Then let's chat! 

The interview will take 60 minutes and cover the following topics:

- Strengths and weaknesses of Flutter as a multi-platform app building framework
- Workflow frictions
- Code sharing
- Testing
- Any other issues you might have run into

We may ask you to demo some of the problems you have encountered in the IDE to get a clear idea.

Please fill out this form if your are interested! https://forms.gle/CjVZdv2VCjt43umh6

**Comment by jackkim9** [comment]
Please also feel free to tag anyone in the comments!

**Comment by MrCsabaToth** [comment]
How broad/public is this study? If I'd refer the form in some Slack channels you'd be like
![fishing_ddos](https://user-images.githubusercontent.com/744949/141225155-225a72b1-b584-44df-8e4d-e6c1ea175d1c.gif)

**Reply by InMatrix** [reply]
Feel free to share this in other channels. Thanks!
